dotnetv3/CloudWatch/Actions/HelloCloudWatch.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+// snippet-start:[CloudWatch.dotnetv3.HelloCloudWatch]
+using Microsoft.Extensions.Hosting;
+using Amazon.CloudWatch;
+using Amazon.CloudWatch.Model;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace CloudWatchActions;
+
+public static class HelloCloudWatch
+{
+    static async Task Main(string[] args)
+    {
+        // Use the AWS .NET Core Setup package to set up dependency injection for the Amazon CloudWatch domain registration service.
```

Comment:
No changes. These are all good, because they're consistent directives and make sense in this context.

Reply:
Thank you for the helpful examples.